src/main/scala/za/co/absa/commons/json/format/JavaTypesSupport.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.commons.json.format
+
+import org.json4s.Formats
+import org.json4s.ext.JavaTypesSerializers
+
+trait JavaTypesSupport extends FormatsBuilder {
+  abstract override protected def formats: Formats = super.formats ++ JavaTypesSerializers.all
```

Comment:
It's called [Stackable traits](https://www.artima.com/scalazine/articles/stackable_trait_pattern.html)
`abstract override` is a special keyword combination in Scala that allows for chaining the method call based on an actual type linearization, instead of the object hierarchy.